cpu/cc2538/periph/uart.c

```diff
@@ -61,26 +61,23 @@ enum {
 #define DIVFRAC_MASK          ( (1 << DIVFRAC_NUM_BITS) - 1 )
 
 /** @brief Indicates if there are bytes available in the UART0 receive FIFO */
-#define uart0_rx_avail() ( UART0->cc2538_uart_fr.FRbits.RXFE == 0 )
+#define uart0_rx_avail() ( uart_config[0].dev->cc2538_uart_fr.FRbits.RXFE == 0 )
```

Comment:
style: either use capitalized define names here, or use static inline functions...

BUT: these 4 functions are not used in the code, so drop them!